### ISSUE TITLE
Add class method to locate soil surface

### ISSUE THREAD

**Issue by m9brady** [m9brady/SMP_to_CSV#6]
Useful for filtering out any weird measurements if the SMP instrument reaches the soil surface

**Comment by kingjml** [comment]
I use this general approach in my R scripts:
If overloaded (max force in header?) back off until first measurement with > 100% change in force,
else if depth ~ 120 cm set a flag for no bottom (ie. full profile without reaching the base),
else back it off 50 mm to avoid any soil.

**Comment by m9brady** [comment]
@kingjml Is it more useful for pick_surf() to return the actual depth value, the array index of the value, or both? 

Reason I ask is that when a user accesses the SMP.snowSurf attribute, giving them the array index might confuse them (not to mention the whole SMP.data vs SMP.subset indexing differences), but if you give them the depth value it should be trivial to extract the array index from either data or subset arrays:

`np.where(SMP.data[:,0] == SMP.snowSurf)[0][0]` 

could also return a dict but this won't address the SMP.data vs SMP.subset issue:

`return { 'val' : 1199.9958174773492, 'idx' : 290399 }`

**Comment by kingjml** [comment]
You're right, returning the idx makes it an issue if you are not aware which data source you are working with.

Return both but make the depth the the primary in case the user does not provide variables for both. I worry about rounding issues as I'm not sure what the decimal precision of the header information is like between SMP versions.
